serve/graph/resolver.go

```diff
@@ -1,4 +1,4 @@
-//go:generate go run github.com/99designs/gqlgen generate
+//go:generate go run github.com/99designs/gqlgen@v0.17.45 generate
```

Comment:
Do we want to fixate the version?

cc @ajnavarro

Reply:
if the version is not pinned, the CI can break without changes if the gen changes

we could pin it in go.mod/go.sum via some kind of "tools.go" also

Reply:
In this file -> https://github.com/gnolang/tx-indexer/blob/main/serve/graph/subscription.resolvers.go#L5
the tool version is part of the output so even on patch version change there will be a diff